src/projectm.c

```diff
@@ -84,6 +93,10 @@ projectm_handle projectm_init(GstProjectM *plugin)
     if (plugin->preset_duration > 0.0)
     {
         projectm_set_preset_duration(handle, plugin->preset_duration);
+
+        // kick off the first preset
+        if (projectm_playlist_size(playlist) > 0 && ! plugin->preset_locked)
+        projectm_playlist_play_next(playlist, true);
```

Comment:
This might just be my personal preference...I always use curly brackets for blocks to make the code more readable

Reply:
done